docs/specification/draft/basic/utilities/tasks.mdx

```diff
@@ -0,0 +1,808 @@
+---
+title: Tasks
+---
+
+<div id="enable-section-numbers" />
+
+<Info>**Protocol Revision**: draft</Info>
+
+<Note>
+
+Tasks are newly-introduced in this version of the MCP specification and are currently considered **experimental**.
+The design and behavior of tasks may evolve in future protocol versions.
+
+</Note>
+
+The Model Context Protocol (MCP) allows requestors (which can be either clients or servers, depending on the direction of communication) to augment their requests with **tasks**. Tasks are durable state machines that carry information about the underlying execution state of the request they wrap, and are intended for requestor polling and deferred result retrieval. Each task is uniquely identifiable by a receiver-generated **task ID**.
+
+## User Interaction Model
+
+Tasks are designed to be **application-driven** - receivers tightly control which requests (if any) support task-based execution and manage the lifecycles of those tasks; meanwhile, requestors are responsible for augmenting requests with tasks and for polling for the results of those tasks.
```

Comment:
`application-driven` - should this be `requestor-driven`? If my understanding is correct, both in the client -> server case and in the server -> client case it's the actually the requestor that decides whether to initiate a task vs. a blocking call.